README.md

```diff
@@ -1,6 +1,10 @@
 css
```

Comment:
How about setting it to `gorilla/css` just to maintain uniformity? 
```
# gorilla/css
```

Reply:
updated in current push, thanks!